### PR TITLE
For #13765 - Fix TopSites item sizes

### LINKED ISSUE #13765: Top sites are more prominent on the homescreen
Based on a large number of user feedback, a lot of our users are finding it read the title of top sites as they are too small. Especially when a user have 2 links saved from the same domain (similar favicon) they find it difficult to distinguish between the two. Users have also expressed that the top site icons are too small for something that is supposed to help them get to where they need faster, whereas the "collections" CTA/placeholder is much larger in proportion. 

## User Story
- As a user, I want my top sites to look more prominent on my homescreen so they are more useful to me.

### Acceptance Criteria
- I can easily read the title of a top sites and distinguish between two top sites from the same domain.
- The size of the top site icons are larger and more prominent than the collections CTA/placeholder. 

### UX
- Specs for the top sites section, and for the updated favicon below
- The Top Sites section should have 4 top sites in a row. This should also allow the full 'Top Articles' text to be visible for the default Pocket top site.
- The favicon size should be increased to 32x32px, leaving 4px of padding inside the favicon container

**Top Sites**
Home - Spec

**Top Site Favicon**
Favicon

### PR DESCRIPTION
**Pull request by codrut-topliceanu**
Fixes #13765 

### Pull Request checklist

- [x] **Tests**: This PR doesn't include any tests as it is only a minor change.
- [x] **Screenshots**: This PR includes screenshots.
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md).

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

I followed the specs in #13765 as best as I could, however the distance between columns needs to be variable(X value in the screenshot) otherwise wider devices will only have top sites on the left side of their screens. I have also added 12dp spacing between rows as it felt appropriate. You can find further screenshots from a Pixel 3XL (for both orientations) below, please let me know if any more changes are required. 
Screenshot 2020-08-27 at 12 14 15

device-2020-08-27-123731

### CODE REVIEW

**Comment by liuche** [comment]
Hi @codrut-topliceanu do we think this could be done in time to make the "late" cut on Tuesday? thank you!

**Comment by codrut-topliceanu** [comment]
I've made the requested changes. @apbitner could you please take a look and tell me if they are ok? (I've also tested this on ldpi and xhdpi emulators/devices and everything seems in order)

landscape top sites

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -614,6 +614,14 @@
         <item name="behavior_halfExpandedRatio">0.4</item>
     </style>
 
+    <style name="Mozac.Widgets.TopSite.Favicon" parent="Mozac.Widgets.Favicon">
```
Comment (Mugurell) [inline app/src/main/res/values/styles.xml:617]: IMO mozac is still kept in resources names only when we intend to overwrite the value from AC.
Here we extend an AC style, but the above need does not exist.
Could we make it our own style with a simpler name, without "mozac"? Thoughts?

**Review by Mugurell — APPROVED** [review]
Looks good, thanks!

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (gabrielluong) [reply app/src/main/res/values/styles.xml]: Agreed with the above. I think it probably makes sense to drop the Mozac to something simpler in Fenix, and only keep to Mozac naming within AC.

Comment (gabrielluong) [reply app/src/main/res/values/styles.xml]: I think this is resolved.

**Comment by softvision-miralobontiu** [comment]
Hi all,
I've noticed on the latest Beta 81.1.0-beta.1, and on 9/2 Nightly, on the Samsung Galaxy Tab S6 (Android 9), and on Samsung Galaxy Tab S3 (Android 8) that the default 3 top sites are not centralized. Is this expected?
Thanks,

Untitled

**Comment by Mugurell** [comment]
They were already aligned to the left.
This patch changed nothing in this regard.

**Pull request merged**